lib/ui/page/home/page/my_profile/crop_avatar/view.dart

```diff
@@ -0,0 +1,135 @@
+// Copyright © 2022-2024 IT ENGINEERING MANAGEMENT INC,
+//                       <https://github.com/team113>
+//
+// This program is free software: you can redistribute it and/or modify it under
+// the terms of the GNU Affero General Public License v3.0 as published by the
+// Free Software Foundation, either version 3 of the License, or (at your
+// option) any later version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+// FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License v3.0 for
+// more details.
+//
+// You should have received a copy of the GNU Affero General Public License v3.0
+// along with this program. If not, see
+// <https://www.gnu.org/licenses/agpl-3.0.html>.
+
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+
+import '/api/backend/schema.dart' show CropAreaInput, PointInput;
+import '/l10n/l10n.dart';
+import '/themes.dart';
+import '/ui/widget/modal_popup.dart';
+import '/ui/widget/widget_button.dart';
+import '/util/platform_utils.dart';
+import 'controller.dart';
+import 'widget/image_cropper/enums.dart';
+import 'widget/image_cropper/widget.dart';
+
+/// View for cropping avatar image.
+///
+/// Intended to be displayed with [show] method.
+class CropAvatarView extends StatelessWidget {
```

Comment:
Increased the size to make it more UI/UX friendly. Let me know if you would like any specific change to enhance it more.
![image](https://github.com/user-attachments/assets/8cec2e87-2788-458d-9f71-28d82cf7ac65)

Reply:
@pratham-sarankar, wouldn't a really large modal window be more UX friendly? When you're cropping your avatar, you'd want to see it as big as possible, so you can position the cropping area more precisely, don't you think? Might be a bad idea - let me know, if you see any UX problems with that.

Reply:
@SleepySquash 

Let me know what do you think about this?

![image](https://github.com/user-attachments/assets/3523398a-b1e5-428f-85f6-8ca101cab263)

Reply:
@pratham-sarankar, `Cancel` and `Done` buttons seem to be too distinct from each other. And these white bars to the left and to the right to the image look a bit too large. What do you think is the best UI/UX for this modal? It's up to you after all.

Reply:
@SleepySquash

In my opinion, It would look better if the width of the `ModalPopup` is equal to the image size. So, we won't have the white bars and `Cancel` and `Done` buttons would not be too distinct.

However, width of the `ModalPopup` is controlled by `modalConstraints.maxWidth`  which cannot be rebuilt. It can be achieved only if I use `ModalPopup` directly (without `ModalPopup.show` method). 

But as you know, task description mentions I shouldn't.

Reply:
@pratham-sarankar, I see your point there, that's true. The only thing that can be done here is restraining the maximum width of the insides of the modal so that the buttons aren't too far from each other. But however in this case I agree with you - you may not focus on the UI.